File: OrderViewer/ClientApp/app/responses/single.response.ts
export interface ISingleResponse<TModel> {
    model: TModel;
    message: string;
    didError: boolean;
    errorMessage: string;
}

export class SingleResponse<TModel> implements ISingleResponse<TModel> {
    public model: TModel;
    public message: string;
    public didError: boolean;
    public errorMessage: string;
}

File: Legacy/OrderViewer/Responses/SingleResponse.cs
using System;

namespace OrderViewer.Responses
{
    public class SingleResponse<TModel> : ISingleResponse<TModel>
    {
        public String Message { get; set; }

        public Boolean DidError { get; set; }

        public String ErrorMessage { get; set; }

        public TModel Model { get; set; }
    }
}
